### PR TITLE
Marketplace update

### PR DESCRIPTION
**Pull request by SamTremko**

### CODE REVIEW

**Comment by SamTremko** [comment]
* **#991** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/991?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#984** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/984?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#983** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/983?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#982** <a href="https://app.graphite.dev/github/pr/vexl-it/vexl/982?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @SamTremko and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/mobile/src/components/InsideRouter/components/Header.tsx [hunk 1]
```diff
@@ -32,11 +33,13 @@ function BtcPriceHeader(): JSX.Element {
...
+function Header(): JSX.Element | null {
   const marketplaceLayoutMode = useAtomValue(marketplaceLayoutModeAtom)
 
   if (marketplaceLayoutMode === 'map') {
     return null
   }
   return <BtcPriceHeader />
 }
+
+export default memo(Header)
```
Comment (kaladivo) [inline apps/mobile/src/components/InsideRouter/components/Header.tsx]: If this is not necessary, remove

File: apps/mobile/src/components/InsideRouter/components/MarketplaceScreen/components/BaseFilterDropdown.tsx [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+  return (
+    <Dropdown
+      data={baseFilterDropdownData}
+      value={baseFilter}
+      onChange={(item) => {
+        setBaseFilter(item.value)
+        if (postSelectActions) postSelectActions()
+      }}
+      style={{}}
```
Comment (kaladivo) [inline apps/mobile/src/components/InsideRouter/components/MarketplaceScreen/components/BaseFilterDropdown.tsx:49]: Is this necessary?

File: apps/mobile/src/components/InsideRouter/components/MarketplaceScreen/components/EmptyListPlaceholder.tsx [hunk 3]
```diff
@@ -82,8 +82,8 @@ function EmptyListPlaceholder({refreshing, onRefresh}: Props): JSX.Element {
   const importedContactsCount = useAtomValue(importedContactsCountAtom)
   const filterActive = useAtomValue(isFilterActiveAtom)
   const marketplaceLayout = useAtomValue(marketplaceLayoutModeAtom)
-  const countOfFilteredOffersBeforeLocation = useAtomValue(
-    countOfFilteredOffersBeforeLocationAtom
+  const offersToSeeInMarketplaceCount = useAtomValue(
+    offersToSeeInMarketplaceCountAtom
```
Comment (kaladivo) [inline apps/mobile/src/components/InsideRouter/components/MarketplaceScreen/components/EmptyListPlaceholder.tsx:86]: Check if works correctly when map view is selected

File: apps/mobile/src/state/marketplace/atoms/filteredOffers.ts [hunk 4]
```diff
@@ -204,28 +197,76 @@ export const filteredOffersIncludingLocationFilterAtom = atom((get) => {
...
+)
+export const btcToProductOffersIncludingLocationAtomsAtom = splitAtom(
+  btcToProductOffersIncludingLocationAtom
+)
+export const productToBtcOffersIncludingLocationAtomsAtom = splitAtom(
+  productToBtcOffersIncludingLocationAtom
+)
+export const sthElseOffersIncludingLocationAtomsAtom = splitAtom(
+  sthElseOffersIncludingLocationAtom
+)
```
Comment (kaladivo) [inline apps/mobile/src/state/marketplace/atoms/filteredOffers.ts]: Check if works and is used on map

File: packages/localization/base.json [hunk 5]
```diff
@@ -499,13 +499,20 @@
   "filterOffers.lowestAmount": "Lowest amount",
   "filterOffers.highestAmount": "Highest amount",
   "filterOffers.selectSortingMethod": "Select sorting method",
-  "filterOffers.searchByText": "Search by text",
```
Comment (kaladivo) [inline packages/localization/base.json:503]: Create issue with no longer used localizations so we can remove those values later

**Review by kaladivo — APPROVED** [review]

**Comment by SamTremko** [comment]
### Merge activity

* **Apr 24, 9:43 AM EDT**: @SamTremko merged this pull request with [Graphite](https://app.graphite.dev/github/pr/vexl-it/vexl/991).

**Pull request merged**